### DISCUSSION TITLE
Wie kann ich das Eigenschaften-Fenster schmaler machen?

### DISCUSSION THREAD

**Discussion by henningTheDev** [SemTalkOnline/SemTalkOnline#11] [Q&A] [answered]
Kann man das Eigenschaften-Fenster schmaler ziehen, damit es nicht so viel von der Zeichnung verdeckt?

Grüße

**Comment by SemTalkOnline** [comment, marked as answer]
Hallo,

die Größe des Eigenschaften Fenster kann derzeit nicht verändert werden.
Es ist jedoch möglich über die Optionen (Zahnrad rechts oben) einzustellen, dass die Eigenschaften nicht als angedocktes Panel sondern als Dialogfenster geöffnet werden kann. Dieses Dialogfenster ist flexibel verschiebbar.
![SemOnOptionenProp](https://user-images.githubusercontent.com/105418212/191233506-6d82b362-dda2-42fe-a14e-883d8ea95d40.PNG)
